### PR TITLE
Fix: DE41706 fix introduced regression

### PR DESCRIPTION
**Pull request by AntonBazhal**

### CODE REVIEW

File: components/d2l-work-to-do/d2l-work-to-do.js [hunk 1]
```diff
@@ -475,7 +475,12 @@ class WorkToDoWidget extends EntityMixinLit(LocalizeWorkToDoMixin(LitElement)) {
 	_itemLoaded(event) {
 		if (this._initialLoad) {
 			this._loadedElements.push(event.target);
-			if (this._loadedElements.length === this._overdueActivities.length + this._upcomingActivities.length) {
+
+			const expectedLoadedActivities = this.fullscreen
+				? this._overdueActivities.length + this._upcomingActivities.length
+				: this._overdueDisplayLimit + this._upcomingDisplayLimit;
```
Comment (surags) [inline components/d2l-work-to-do/d2l-work-to-do.js:481]: If not fullscreen this should probably be `min(this._overdueDisplayLimit, this._overdueActivities.length) + min(this._upcomingDisplayLimit, this._upcomingActivities.length)` to account for the scenario where there aren't enough activities

Comment (AntonBazhal) [reply components/d2l-work-to-do/d2l-work-to-do.js:481]: Ah, I think it actually needs to be `min(Constants.MaxWidgetDisplay, this._overdueActivities.length + this._upcomingActivities.length)`

**Review by JasperAtD2L — APPROVED** [review]

File: components/d2l-work-to-do/d2l-work-to-do.js (hunk 1 above)
Comment (JasperAtD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js]: I'm wondering if we should just get the value from the templates: https://github.com/BrightspaceHypermediaComponents/activities/pull/1449/files#diff-d300a684213152d82078d4d614f2f7ede435053c312f283cc93533dbae942e94R192 so we know exactly how many sub components are on screen.

Comment (JasperAtD2L) [reply components/d2l-work-to-do/d2l-work-to-do.js]: Actually nvm the render gets called a bunch of times, this should be good.

**Pull request merged**